var/spack/repos/builtin/packages/py-theano/package.py

```diff
@@ -39,7 +39,7 @@ class PyTheano(PythonPackage):
 
     depends_on('python@2.6:2.8,3.3:')
 
-    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools', type='build', 'run')
```

Comment:
Is this interpreted correctly? I mean:
```python
depends_on('py-setuptools', type='build', 'run')
```
instead of:
```python
 depends_on('py-setuptools', type=('build', 'run'))
```